css-audit.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	format: 'html',
+	all: true,
+	filename: 'wp-admin',
+	audits: [ [ 'property-values', 'font-size' ] ],
```

Comment:
I think I'd rather leave the `all` property out of the config file - it was intended as a shortcut for the CLI command, but in a config file I think it would be better to list out each audit intentionally.

This doesn't need any special handling, I think we can just remove it from this file.